metagraph/tests/annotation/test_converters.cpp

```diff
@@ -25,15 +25,20 @@ const std::string test_dump_basename_row_compressed_to_rowflat = test_dump_basen
 class ConvertFromRowCompressed : public ::testing::Test {
   protected:
     static const uint64_t num_rows = 5;
-    static RowCompressed<> *initial_annotation;
-    static MultiLabelEncoded<std::string> *annotation;
+    RowCompressed<> *initial_annotation;
+    MultiLabelEncoded<std::string> *annotation;
+    graph::DBGSuccinct *graph = nullptr;
 
     virtual void SetUp() {
         initial_annotation = new RowCompressed<>(num_rows);
-        initial_annotation->add_labels({ 0 }, {"Label0", "Label2", "Label8"});
-        initial_annotation->add_labels({ 2 }, {"Label1", "Label2"});
-        initial_annotation->add_labels({ 3 }, {"Label1", "Label2", "Label8"});
-        initial_annotation->add_labels({ 4 }, {"Label2"});
+        initial_annotation->add_labels({ 0 }, { "Label0", "Label2", "Label8" });
+        initial_annotation->add_labels({ 2 }, { "Label1", "Label2" });
+        initial_annotation->add_labels({ 3 }, { "Label1", "Label2", "Label8" });
+        initial_annotation->add_labels({ 4 }, { "Label2" });
+
+        graph = new graph::DBGSuccinct(3);
+        graph->add_sequence("ACGTCAC");
+        graph->mask_dummy_kmers(1, false);
```

Comment:
Why did you change this test? Please add new tests but don't change the old ones, unless it's some kind of refactoring. But this change actually makes a conceptual difference.

In practice, we never mask out the dummy k-mers in our graphs. This is a legacy feature.

Reply:
I didn't change the test, in the sense that the previous tests are not in any way affected by my change. I simply added a graph to the test so that we can also test row-diff annotations.

Reply:
I think the graph had already been there, and I guess used by other annotations as well. You added the mask, which changed the graph, so effectively changed the tests too. Did I miss something?

Reply:
Yes, you missed the fact that I added the graph :)

Reply:
oh, indeed. For some reason, I thought the graph had been there.